app/controllers/secured_controller.rb

```diff
@@ -0,0 +1,11 @@
+class SecuredController < ApplicationController
```

Comment:
Don't you think the code would have been simpler had we just place all this logic in ApplicationController instead of having a separate controller where all other controllers inherit from?

Reply:
Optional:

I still wonder why we didn't just simply use ApplicationController to house the logic in this file.